skbio/io/format/biom.py

```diff
@@ -28,13 +30,19 @@
 
 Examples
 --------
-Here we will write an existing BIOM table, and re-read it. Note that the Table
-from ``biom`` implicitly gets the ``.write`` method from the IO registry.
-
->>> import biom, skbio
->>> biom.example_table.write('test.biom')
-'test.biom'
->>> roundtrip = skbio.read('test.biom', into=skbio.Table)
+Here we're demonstrating how to convert a BIOM table into HDF5 format and then
+read it back.
```

Comment:
Can add a sentence: "This `ByteIO` object can be a file path in a regular use case."